integrations/anthropic/README.md

```diff
@@ -0,0 +1,40 @@
+# anthropic-haystack
+
+[![PyPI - Version](https://img.shields.io/pypi/v/amazon-bedrock-haystack.svg)](https://pypi.org/project/amazon-bedrock-haystack)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/amazon-bedrock-haystack.svg)](https://pypi.org/project/amazon-bedrock-haystack)
+
+-----
+
+**Table of Contents**
+
+- [Installation](#installation)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Installation
+
+```console
+pip install anthropic-haystack
+```
+
```

Comment:
We could add a section about the examples here:
```
## Examples
You can find a code example showing how to use the Generator under the `example/` folder of this repo.
````

And that example folder is missing at the moment. We need to add it.